### DISCUSSION TITLE
stuck on signing into full node

### DISCUSSION THREAD

**Discussion by DABBATHEHUTT** [Chia-Network/chia-blockchain#17366] [Support]
after the last update i wont go passed the signing into node spot.  very newbie.  what do i check first for better help? looked up how to do a v2 db update. unsure whats the correct what to do it.

**Comment by BrandtH22** [comment]
Hey @DABBATHEHUTT , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

It sounds like you are on the right path with the db upgrade. There are two methods to complete this based on how much space you have available on the drive that has the db (generally the OS drive is left at defaults). If you have more than ~ 300GB available on that drive then you can use the cli upgrade option, if not then you want to use the torrent snapshot option:

Steps for the CLI command:
1. Stop the chia client (this can be done while chia is running but stopping the client saves from any potential issues)
2. Run the command `chia db upgrade` (you do not need the input or output paths if these are still default)
- This process will take some time and requires additional space as the v1 db is compacted into the v2 db which leaves the v1 db intact
3. Verify the config file (~\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
4. After the above completes start the chia client
5. If all loads up properly (might take upwards of 30 minutes to load) then you can safely delete the v1 db found here (~\.chia\mainnet\db\blockchain_v1_mainnet.sqlite)


Steps for using the snapshot torrent: 
1. Stop chia
2. Delete the db at ~\.chia\mainnet\db\ (you can delete all files in that db directory)
3. Download the torrent file from https://www.chia.net/downloads/#database-checkpoint
4. Use a torrent client to download the full db
5. Unpack/reassemble the torrent file that was downloaded (on windows I used 7zip, Mac and linux have built in tools that work for this)
6. Move the db to the correct folder (~\.chia\mainnet\db\) and update the name to "blockchain_v2_mainnet.sqlite" (removing the date information in the name)
7. Verify the config file (~\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
8. Launch chia and wait for a bit (the height to hash and peers files need to be built so this can take upwards of 30 minutes)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
